code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm

```diff
@@ -166,4 +166,11 @@
 	description = "An identification card given to free IPC residents within the Republic of Biesel."
 	path = /obj/item/clothing/accessory/badge/passcard/synthetic
 	cost = 0
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsian mantle"
+	description = "An uniform mantle identifying Hephaestus IPC's from Burzsia. Operation history and specifications are printed underneath the tarp."
+	path = /obj/item/clothing/accessory/poncho/IPC_Mantle
```

Comment:
```suggestion
	path = /obj/item/clothing/accessory/poncho/ipc_mantle
```
Technically not a hard and fast rule I don't think but the vast vast majority of items use all lowercase paths